exhibits/app.js

```diff
@@ -0,0 +1,50 @@
+import 'babel-polyfill';
```

Comment:
What's the `babel-polyfill` used for here? Maybe a couple of comments to explain its use would help with understanding.